h/static/scripts/tags.coffee

```diff
@@ -14,24 +14,24 @@ createTagHelpers = ['localstorage', (localstorage) ->
 
   # Add newly added tags from an annotation to the stored ones and refresh
   # timestamp for every tags used.
-  storeTags: (tags) ->
-    savedTags = localstorage.getObject TAGS_MAP_KEY
+  store: (tags) ->
+    savedTags = localStorage.getObject TAGS_MAP_KEY
     savedTags ?= {}
 
     for tag in tags
-      if savedTags[tag.text]?
+      if savedTags[tag]?
```

Comment:
Ahhhh, sorry. I'd misread. Tags are guaranteed to just be strings now. That makes more sense. My point about backwards compatibility still stands, however.